powershell-adapter/powershell.resource.ps1

```diff
@@ -1,427 +1,349 @@
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
-
 [CmdletBinding()]
 param(
-    [ValidateSet('List','Get','Set','Test','Export','Validate')]
-    $Operation = 'List',
-    [Switch]
-    $WinPS = $false,
-    [Parameter(ValueFromPipeline)]
-    $stdinput
+    [Parameter(Mandatory = $true, Position = 0, HelpMessage = 'Operation to perform. Choose from List, Get, Set, Test, Export, Validate.')]
+    [ValidateSet('List', 'Get', 'Set', 'Test', 'Export', 'Validate')]
+    [string]$Operation = 'Default',
+    [Parameter(Mandatory = $false, Position = 1, ValueFromPipeline = $true, HelpMessage = 'Configuration or resource input in JSON format.')]
+    [string]$stdinput = '@{}'
```

Comment:
Would be better to call this `$jsonInput` as it's not always being read from STDIN.  Also, may be better to not initialize it to empty JSON and instead check for empty string later.